code/modules/vehicles/multitile/hardpoints.dm

```diff
@@ -278,7 +278,7 @@ Currently only has the tank hardpoints
 	log_combat(usr, usr, "fired the [src].")
 	log_explosion("[usr] fired the [src] at [AREACOORD(loc)].")
 	P.fire_at(T, owner, src, P.ammo.max_range, P.ammo.shell_speed)
-	if(!CONFIG_GET(flag/tank_mouth_noise))
```

Comment:
disagree with combining these, either commit to removing the joke noises or keep it as a separate thing

Reply:
I disagree with this.

Reply:
To be fair if we consider the possible usage, then individual configs is better, as it gives the admins the choice of what to enable and what not to.

Reply:
It has never been used and I sure hope it never will. This simply doesn't fit our server and having 200 configs for every little thing is not really feasible nor maintainable.

Reply:
Eh, remove both then.

Reply:
Done